### DISCUSSION TITLE
Is it possible to reconcile the application and the set of applications separately?

### DISCUSSION THREAD

**Discussion by zeusal** [argoproj/argo-cd#17782] [Q&A] [answered]
Hello,

I have that question, Is it possible to reconcile application and application set separately?

I need to reconcile application set separately, i.e. I need to reconcile application with git every 10 minutes and application set every 5 minutes, or update application set via API/CLI, is it possible?

Regards!!

**Comment by crenshaw-dev** [comment, marked as answer]
The Application reconciliation time is defined here: https://github.com/argoproj/argo-cd/blob/4abc9929281296122db0eace1607583187dd9944/docs/operator-manual/argocd-cm.yaml#L309-L312

ApplicationSets' reconciliation timeouts are generator-specific. The git generator reconciles every three minutes by default, but you can override that in the ApplicationSet definition: https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Generators-Git/

**Reply by zeusal** [reply]
Hello @crenshaw-dev  thank you for your answer!

The webhook option should solve my need. However, in the future do you plan to allow refreshing application set from the api or the cli?

Regards!
